plugin/testdata/unauthorized-type.yml

```diff
@@ -28,7 +28,7 @@ trigger:
 
```

Comment:
tests panic if i put a terminator here `...`

Reply:
Yaml loader is now handling this kind of terminator too, adding it to the test file.